source/javascripts/modules/anchored-headings.js

```diff
@@ -0,0 +1,19 @@
+(function($, Modules) {
+  'use strict';
+
+  Modules.AnchoredHeadings = function() {
+    this.start = function($element) {
+      var headings = $element.find('h1, h2, h3, h4, h5, h6');
+      headings.each(injectAnchor);
+    };
+
+    function injectAnchor() {
+      var $this = $(this);
+      var fragment = '#' + $this.attr('id');
```

Comment:
Can save a variable and a concatenation with:

```js
$this.prepend(
    '<a href="#' + $this.attr('id') + '" class="anchored-heading__icon" aria-hidden="true">🔗</a>'
);
```

Unless you think the extra step aids legibility?